src/Host/Client/Impl/Session/RSession.cs

```diff
@@ -103,24 +105,36 @@ internal sealed class RSession : IRSession, IRCallbacks {
 
         public Task<IRSessionInteraction> BeginInteractionAsync(bool isVisible = true, CancellationToken cancellationToken = default(CancellationToken)) {
             if (!_isHostRunning) {
-                return CanceledInteractionTask;
+                return CanceledBeginInteractionTask;
             }
 
             RSessionRequestSource requestSource = new RSessionRequestSource(isVisible, _contexts, cancellationToken);
```

Comment:
Okay. Since this isn't part of this change, can you file a bug? And I'll address it in a separate PR to keep things clean.